entrypoint/entrypoint.sh

```diff
@@ -60,6 +73,12 @@ print_title(){
     echo
 }
 
+print_env_vars(){
+    print_title "Enviornment Variables"
+    printf "\n${GREEN} OPERATOR_BUNDLE: $OPERATOR_BUNDLE ${ENDCOLOR}\n"
+    printf "\n${GREEN} OCP version: $OCP_VERSION ${ENDCOLOR}\n"
+}
+
 # Deploy UIPlugins and Observability Operator based on OpenShift Version 
 if [[ $(bc <<< "$OCP_VERSION == 4.11") -eq 1 ]]; then
```

Comment:
I think we discussed that this part is not needed as the Operator knows which plugins can be installed but we want developers to have the option to check which plugins are installed rather than installing everything.

Reply:
I update the Makefile to add commands to deploy plugins separately in the latest commit.